modules/sdk/project-templates/project-templates-extensions/src/main/java/com/liferay/project/templates/extensions/util/ProjectTemplatesUtil.java

```diff
@@ -160,6 +160,12 @@ public static String getArchetypeVersion(String artifactId)
 		return projectTemplateJarVersionsProperties.getProperty(artifactId);
 	}
 
+	public static String getLiferayVersion() {
```

Comment:
Hey @lawrence-lee  lets not add this method, as it isn't exactly clear what this should be used for.  Instead in the test case, just locally declare a local variable for liferayVersion.